e2e/package.json

```diff
@@ -1,28 +1,34 @@
 {
   "devDependencies": {
-    "@testing-library/cypress": "5.0.2",
-    "axios": "0.19.0",
+    "@testing-library/cypress": "5.3.0",
+    "axios": "0.19.2",
     "body-parser": "1.19.0",
-    "cypress": "3.8.0",
-    "cypress-file-upload": "3.5.1",
-    "cypress-multi-reporters": "1.2.3",
-    "cypress-plugin-tab": "1.0.4",
+    "cypress": "4.0.2",
+    "cypress-file-upload": "3.5.3",
+    "cypress-multi-reporters": "1.2.4",
+    "cypress-plugin-tab": "1.0.5",
     "cypress-wait-until": "1.6.1",
     "express": "4.17.1",
     "lodash": "4.17.15",
     "merge-deep": "3.0.2",
-    "mocha": "6.2.2",
+    "mocha": "7.0.1",
     "mocha-junit-reporters": "1.23.6",
     "mocha-multi-reporters": "1.1.7",
     "mochawesome": "4.1.0",
-    "mochawesome-merge": "2.1.0",
-    "mochawesome-report-generator": "4.0.1",
-    "start-server-and-test": "1.10.6"
+    "mochawesome-merge": "4.0.1",
+    "mochawesome-report-generator": "4.1.0",
+    "start-server-and-test": "1.10.8"
   },
   "scripts": {
-    "cypress:run": "cypress run --config ignoreTestFiles='**/enterprise/**/*.js'",
     "cypress:open": "cypress open",
-    "cypress:run:enterprise": "cypress run --spec='cypress/integration/enterprise/**/*_spec.js'",
```

Comment:
Can we have it by default to run it on chrome, i.e. the command `cypress:run` and `cypress:run:enterprise` should by default be running on Chrome as usual.

Reply:
Updated